src/main/java/com/_up/megastore/services/implementations/ProductService.java

```diff
@@ -56,5 +57,36 @@ private String saveProductImage(MultipartFile multipartFile) {
     private Product getVariantOf(UUID variantOfId) {
         return variantOfId != null ? findProductByIdOrThrowException(variantOfId) : null;
     }
+    @Override
+
+
+
+    public ProductResponse updateProduct(UUID productId, UpdateProductRequest updateProductRequest, MultipartFile multipartFile){
```

Comment:
No te va a dar un error, pero se ponen las etiquetas justamente previo a lo que se desea aplicarle ese label

```suggestion
    @Override
    public ProductResponse updateProduct(UUID productId, UpdateProductRequest updateProductRequest, MultipartFile multipartFile){
```